lib/cn/kvset.c

```diff
@@ -2422,7 +2456,8 @@ kvset_iter_free_buffers(struct kvset_iterator *iter, struct kblk_reader *kr)
     vlb_free(kr->kr_buf[1].kmd_buf, kr->kr_buf[1].kmd_used_sz);
```

Comment:
What is the significance of the 0s and 1s used here? (Not part of the review)

Reply:
Something related to double buffering? Not sure.